run_dir/static/js/fc_trend_plot.js

```diff
@@ -21,12 +21,33 @@ function refresh_plot(){
 }
 
 function make_plot(key, name, display_by, filter_inst_type, filter_inst, color_type, plot_type){
+    function sumBPYield(series) {
+        var sum = 0;
+        if (series && Array.isArray(series)) {
+            for (let i = 0; i < series.length; i++) {
+                if (series[i] && series[i].visible && series[i].data && Array.isArray(series[i].data)) {
+                    for (let j = 0; j < series[i].data.length; j++) {
+                        if (series[i].data[j] && typeof series[i].data[j].bp_yield === 'number') {
+                            sum += series[i].data[j].bp_yield;
+                        }
+                    }
+                }
+            }
+        }   
+        return sum;
+    }    
     var toplot={
         chart: {
-            type: plot_type
+            type: plot_type,
+            events: {
+                render: function(){
+                    var formatted_sum = (sumBPYield(this.series)).toLocaleString();
+                    this.setTitle({text: 'Accumulated yield in Mbp: ' + formatted_sum}, false, false);
+                }
+            }
         },
         title: {
-            text : name+' of the recent flowcells'
+            text: function() { return name+' of the recent flowcells, yield sum in Mbp: ' +  sumBPYield(series, data, true); }
```

Comment:
`sumBPYield(series, data, true)` only takes a single argument in the definition above?

Reply:
Right, it's old, from before 👍🏻